cpu-bind=MASK - bp1-compute180, task  0  0 [3807247]: mask 0x7ffe00007ffe set
Traceback (most recent call last):
  File "/user/home/zj22662/photon-net/reup_vary_new.py", line 162, in <module>
    data_prod_iterator(varied_list, global_var_name, is_key, log_file, folder, file_indent, start_idx)
  File "/user/home/zj22662/photon-net/reup_vary_new.py", line 113, in data_prod_iterator
    g.input_config = g.input_config_maker(g.input_positions, g.num_modes_circ, g.p_suc_inputs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/zj22662/photon-net/p_pack/globals.py", line 179, in input_config_maker
    arr[int(idx)] = 1
    ~~~^^^^^^^^^^
IndexError: list assignment index out of range
